main.go

```diff
@@ -48,6 +49,11 @@ var (
 	metricsAddr          string
 	enableLeaderElection bool
 	probeAddr            string
+	// WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
+	// which specifies the Namespace to watch.
+	// An empty value means the operator is running with cluster scope.
+	watchNamespaceEnvVar = "WATCH_NAMESPACE"
```

Comment:
Why is it WATCH_NAMESPACE and not just NAMESPACE? Is there any other namespace we are configuring in the operator?